vic/drivers/image/src/vic_init.c

```diff
@@ -57,6 +57,7 @@ vic_init(void)
     size_t                     i;
     size_t                     j;
     size_t                     k;
+    size_t                     m;
```

Comment:
@jhamman: So it is still not clear to me why this variable `m` was added. If it is not used it should not be there. After all we spent a long clean up getting rid of these. Please clarify.